src/app.js

```diff
@@ -0,0 +1,275 @@
+class FCPXML {
```

Comment:
consider adding "use strict" directive?

Reply:
It is already
[](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#strict_mode_for_classes)